### PR TITLE
Tempban Implement

### PR DESCRIPTION
**Pull request by ghost**
Making tban and /noob supporting reasons, oblivious i made the reasons hiding if the /noob dont have the custom reason.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java [hunk 1]
```diff
@@ -14,49 +14,34 @@
...
+        if (args.length > 1) {
+            reason = StringUtils.join((Object[])args, " ", 1, args.length);
         }
-        else
-        {
+        else {
             reason = "You have been temporarily banned for 5 minutes.";
         }
-
-        // strike with lightning effect:
```
Comment (ItzLevvie) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java]: Again.. what was the reason for this removal?

**Comment by Wild1145** [comment]
Thank you for contributing to the project, could you please amend your opening message to include the issue that this PR resolves. Thank You.

**Comment by marcocorriero** [comment]
@Wild1145 it adds in tempban the feature to say the reason in /noob

**Comment by Wild1145** [comment]
@marcocorriero Please read my previous message on this PR.

**Comment by marcocorriero** [comment]
#1835

**Comment by ItzLevvie** [comment]
Needs to be formatted to the correct style.

**Comment by marcocorriero** [comment]
For me works great

**Comment by Wild1145** [comment]
@marcocorriero Again, please see my previous comment and instructions in said comment. It makes life a whole lot easier.

**Comment by Wild1145** [comment]
This also required proper formatting as noted above before we can consider this for merging.

**Comment by marcocorriero** [comment]
@Wild1145 this works fine on mf, and what you mean "formatting"?

**Comment by Wild1145** [comment]
@marcocorriero Read the contributing guidelines.

**Pull request closed without merging**